src/main/java/rs117/hd/HdPluginConfig.java

```diff
@@ -162,17 +161,18 @@ default Contrast contrast()
 		return Contrast.DEFAULT;
 	}
 
+	@Range(
+		min = 1,
+		max = 50
+	)
 	@ConfigItem(
-		keyName = "brightness",
+		keyName = "brightness2",
```

Comment:
Maybe a better name here than `brightness2`, like `detailedBrightness` or something of the sort?

Reply:
Yeah well i had to change it or otherwise it will override the old brightness setting and you cant convert it anymore. but the brightness2 only shows in the config file and the method to get it is still called brightness so it doesn't matter that much.